pandas/core/frame.py

```diff
@@ -1043,6 +1041,56 @@ def _repr_html_(self) -> str | None:
         else:
             return None
 
+    @overload
+    def to_string(
+        self,
+        buf: None = None,
+        columns: Sequence[str] | None = None,
+        col_space: int | None = None,
```

Comment:
Not directly related to your pr, but this type hint is wrong, can also be list[int] or dict

Reply:
Ok, I can look into that. Would like to get current version to pass first, then I’ll do this.